src/um/sapiddk.rs

```diff
@@ -0,0 +1,294 @@
+// Copyright © 2017 winapi-rs developers
+// Licensed under the Apache License, Version 2.0
+// <LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your option.
+// All files in the project carrying such notice may not be copied, modified, or distributed
+// except according to those terms.
+use ctypes::{c_float, c_int, c_void};
+use shared::guiddef::CLSID;
+use shared::minwindef::{BOOL, DWORD, ULONG};
+use shared::rpcdce::RPC_IF_HANDLE;
+use um::sapi::*;
+use um::unknwnbase::{IUnknown, IUnknownVtbl};
+use um::winnt::{HRESULT, LPCWSTR, ULONGLONG, WCHAR};
+pub use um::sapiddk51::{
+    SPRECOEXTENSION,
+    SPALTERNATESCLSID,
+};
+pub const SR_LOCALIZED_DESCRIPTION: &'static str = "Description";
+extern {
+    pub static __MIDL_itf_sapiddk_0000_0000_v0_0_c_ifspec: RPC_IF_HANDLE;
```

Comment:
These should just be left out. They're stuff leftover by IDL and don't seem to be useful at all.